wkcuber/tests/scripts/tiff_formats_cubing.sh

```diff
@@ -0,0 +1,78 @@
+set -xe
+mkdir -p testoutput/tiff4
+
+multi_output_files=("test_C" "test_S" "test_CS")
+
+# simplest case: Only XY and I (alternative Z axis name) axis
+python -m wkcuber.convert_image_stack_to_wkw \
+  --jobs 2 \
+  --batch_size 8 \
+  --layer_name color \
+  --max_mag 2 \
+  --target_mag 2\
```

Comment:
I like that you used `--target_mag 2`, but I would also increase the `--max_mag`. Otherwise nothing will be downsampled.